website/static/js/projectSettings.js

```diff
@@ -67,19 +96,45 @@ var NodeCategorySettings = oop.extend(
                 .then(function(response) {
                     return response.updated_fields.category;
                 })
-                .done(self.updateSuccess.bind(self))
-                .fail(self.updateError.bind(self));
+                .done(self.updateCategorySuccess.bind(self))
+                .fail(self.updateCategoryError.bind(self));
         },
+        updateTitle: function() {
+            var self = this;
+            return $osf.putJSON(self.titleDescriptionEditUrl, {
+                    title: $osf.htmlEscape(self.title()),
+                })
+                .done(self.updateDescriptionAfterTitle.bind(self))
+                .fail(self.updateTitleError.bind(self));
+        },
+        updateDescriptionAfterTitle: function() {
```

Comment:
@sloria I kept it as updateDescription at first, but Lyndsy got confused and was wondering why updateDescription was called within updateTitle, so I changed it. 
Think I should change it back?